custom_components/steam_wishlist/util.py

```diff
@@ -30,15 +31,49 @@ def get_steam_game(game_id: int, game: Dict[str, Any]) -> SteamGame:
         # Price is an integer so $6.00 is 600.
         sale_price = round(int(pricing["price"]) * 0.01, 2)
 
+    reviews_percent = game.get('reviews_percent', 'N/A')
+    review_desc = game.get('review_desc', 'No reviews')
+    rating_info = f"Reviews:&nbsp;&nbsp;{reviews_percent}% ({review_desc})"
+
+    tags = game.get("tags", [])
+    tags_string = ", ".join(tags)
+
+    try:
+        original_price = float(normal_price if normal_price is not None else 0)
+        sale_price_val = float(sale_price if sale_price is not None else original_price)
+        discount_percentage = int(discount_pct) if discount_pct is not None else 0
+        if original_price == 0:
+            price_info = "Price:&nbsp;&nbsp;TBD"
+        else:
+            original_price_formatted = f"{original_price:.2f}"
+            if sale_price is not None:
+                strikethrough_price = ''.join(ch + "\u0336" for ch in original_price_formatted[:-1]) + original_price_formatted[-1]
+                price_info = f"{strikethrough_price} ${sale_price_val:.2f} ({discount_percentage}% off)&nbsp;&nbsp;🎫"
+            else:
+                price_info = f"Price:&nbsp;&nbsp;${original_price:.2f}"
+    except (ValueError, TypeError):
+        price_info = "Price information unavailable"
+
+
+    release_date = ("Release date:&nbsp;&nbsp;" + datetime.datetime.utcfromtimestamp(int(game.get("release_date", "0"))).strftime("%b %d, %Y") + "&nbsp;&nbsp;🆕" if datetime.datetime.utcnow() < datetime.datetime.utcfromtimestamp(int(game.get("release_date", "0")) + 86400) else "Released:&nbsp;&nbsp;" + datetime.datetime.utcfromtimestamp(int(game.get("release_date", "0"))).strftime("%b %d, %Y")) if str(game.get("release_date", "0")).isdigit() else "Unknown"
+
     game: SteamGame = {
-        "box_art_url": game["capsule"],
-        "normal_price": normal_price,
-        "percent_off": discount_pct,
+        "title": game["name"],
+        "rating": rating_info,
+        "price": price_info,
+        "genres": ", ".join(game.get("tags", [])),
+        "release": release_date,
+        "airdate": game.get("release_date", ""),
+        "normal_price": str(normal_price),
+        "percent_off": str(discount_pct),
         "review_desc": game.get("review_desc", "No user reviews"),
         "reviews_percent": game.get("reviews_percent", 0),
         "reviews_total": game.get("reviews_total", "0"),
-        "sale_price": sale_price,
-        "steam_id": game_id,
-        "title": game["name"],
+        "sale_price": sale_price if not config_entry.options.get("show_all_wishlist_items", True) else str(sale_price),
```

Comment:
I'm not exactly sure why this is needed.  Why would you want to use `"None"` instead of `None`?

Reply:
Originally, I was looking for the simplest way to display all wishlist items instead of only `on_sale` items.  I realized when switching to "None" instead of `None` for `sale_price`, that effectively did what I wanted; hence why I changed that line.  There is probably a better way to do it; but, I just did whatever was quick and effective.  In any case, after successfully making all the changes necessary to directly use boolean (store_all_wishlist_items) instead of `config_entry` the line looks like this:  `"sale_price": str(sale_price) if store_all_wishlist_items else sale_price,`